docs/fastdds/dds_layer/dds_layer.rst

```diff
@@ -3,6 +3,39 @@
 DDS Layer
 =========
 
+*eProsima Fast DDS* exposes two different APIs to interact with the communication service at different levels.
+The main API is the Data Distribution Service (DDS) Data-Centric Publish-Subscribe (DCPS) Platform Independent Model
+(PIM) API, or :ref:`api_pim_dds_dcps_pim` for short, which is defined by the
+`Data Distribution Service (DDS) version 1.4 specification <https://www.omg.org/spec/DDS/1.4>`_, to which *Fast DDS*
+complies.
+This section is devoted to explain the main characteristics and modes-of-use of this API under *Fast DDS*, providing an
+in depth explanation of the five modules into which it is divided:
+
+* :ref:`dds_layer_core`:
+  It defines the abstract classes and interfaces that are refined by the other modules.
+  It also provides the Quality of Service (QoS) definitions, as well as support for the notification-based interaction
+  style with the middleware.
+
+* :ref:`dds_layer_domain`:
+  It contains the :ref:`dds_layer_domainParticipant` class that acts as an entry-point of the Service,
+  as well as a factory for many of the classes. The :class:`DomainParticipant` also acts as a container for the other
+  objects that make up the Service.
+
+* :ref:`dds_layer_publisher`:
+  It contains the :ref:`dds_layer_publisher_publisher` and :ref:`dds_layer_publisher_dataWriter` classes as well as
+  the :ref:`api_pim_publisherlistener` and :ref:`dds_layer_publisher_dataWriterListener` interfaces, and more
+  generally, all that is needed on the publication side.
```

Comment:
I would give the general description first and the specifics afterwards. Something like:

> It describes the classes used on the publication side, including the :ref:`dds_layer_publisher_publisher` and :ref:`dds_layer_publisher_dataWriter`, as well as the :ref:`dds_layer_publisher_publisherlistener` and :ref:`dds_layer_publisher_dataWriterListener` interfaces

Careful, one of the references link to the API instead of the documentation.

Reply:
done